### DISCUSSION TITLE
Link error in wiki

### DISCUSSION THREAD

**Discussion by axgd-code** [immich-app/immich#548] [Q&A]
Hello !

I would be willing to help with the translation into French.
Unfortunately in https://github.com/immich-app/immich/wiki/Translation-Guide the discord link is invalid.
Would it be possible to correct the link and if you need, add me in "localizely" as a translator?

Thank you !

**Comment by panoti** [comment]
Hi @llexaxell, you can fork this repo, then create French locale file and finally create a pull request on our main branch. You also refer to this existed pull request https://github.com/immich-app/immich/pull/343. FYI, there is no i18n feature on web, but our team will support ASAP.
